File: csharp/SlimDownYourAggregates.Tests/Slimmed/Entities/Translation.cs
namespace SlimDownYourAggregates.Tests.Slimmed.Entities;

public class Translation
{
    public Translation(Language language, Translator translator)
    {
        Language = language;
        Translator = translator;
    }

    public Language Language { get; }
    public Translator Translator { get; }
}

File: java/src/main/java/io/eventdriven/slimdownaggregates/shorter/slimmed/entities/Translation.java
package io.eventdriven.slimdownaggregates.shorter.slimmed.entities;

public class Translation {
  private final Language language;
  private final Translator translator;

  public Translation(Language language, Translator translator)
  {
    this.language = language;
    this.translator = translator;
  }
}
